src/Applications/MoveCountsViewer.py

```diff
@@ -65,34 +65,36 @@
 
 
 class MoveCountsViewer:
-    """A class to describe MoveCountsViewer attributes
+    """ A class to describe MoveCountsViewer attributes
     """
-
-    def __init__(self, input_file_suffix="vom"):
+    def __init__(self, n_processors: int = 0, input_file_name: str = None, input_file_suffix: str = 'out',
```

Comment:
Please use double quotes for strings for style consistency thanks

Reply:
Ok, I can change that, but I use single quotes, because it makes things simple when you need have a string which containg double `"` quotes like html or JSON

Reply:
Yes, we can do this but if so we have to do it **everywhere in LBAF** for consistency. So let's keep the current style for now, which is consistent. (double quotes for non-atomic strings; single quotes for characters including special ones). Let's keep the reformatting to everything single quoted for later.